desktop/NodeManager.ts

```diff
@@ -379,6 +380,35 @@ class NodeManager extends AbstractManager {
     if (this.isNodeRunning()) return true;
     this.sendNodeStatus(DEFAULT_NODE_STATUS);
     this.$_nodeStatus.reset();
+    const isUpToDate = await this.isDatabaseUpToDate().catch((err) => {
+      logger.error('isDatabaseUpToDate', err);
+      // Fallback to default syncing
+      return {
+        synced: true,
+        db: 0,
+        latest: 0,
+      };
+    });
+    logger.log('isDatabaseUpToDate', isUpToDate);
+    if (!isUpToDate.synced) {
+      const prompt = await showGenericPrompt(this.mainWindow.webContents, {
+        title: 'Do you want to quick sync?',
+        message: [
+          'The database is too far behind:',
+          `Latest layer in your database: ${isUpToDate.db}`,
+          `Latest layer in the network: ${isUpToDate.latest}`,
+          '',
+          'Do you want to run a quicksync, which will download database from official Spacemesh storage?',
+        ].join('\n'),
+        confirmTitle: 'Yes, quicksync!',
+        cancelTitle: 'Sync as usual',
+      });
```

Comment:
I'm afraid the users will need some kind of explanation about the pros and cons or differences in general. Otherwise, they will either ask on Discord what is better for them to choose or they will always choose quicksync and therefore the prompt will be useless 🤷‍♀️